go/src/koding/klient/machine/client/supervised.go

```diff
@@ -77,6 +79,19 @@ func (s *Supervised) MountGetIndex(path string) (idx *index.Index, err error) {
 	return
 }
 
+// DiskBlocks calls registered Client's DiskBlocks method and returns its
+// result if it's not produced by Disconnected client. If it is, this function
+// will wait until valid client is available or timeout is reached.
+func (s *Supervised) DiskBlocks(path string) (size, total, free, used uint64, err error) {
```

Comment:
i wasnt aware of `fs.DiskInfo` just proposed DiskStats after `runtime.MemStats` Both LGTM, you choose @ppknap :D